setup.py

```diff
@@ -35,20 +35,21 @@
     "tox>=3.5.2",
     "twine>=1.13.0",
     "wheel>=0.33.1",
+    "lkaccess",
 ]
 
 interactive_requirements = [
     "altair",
     "jupyterlab",
     "matplotlib",
-    "ipykernel"
+    "ipykernel",
+    "lkaccess",
```

Comment:
Should this be in interactive requirements? If you wanted to install multiple groups you could do `pip install fov_processing_pipeline[dev,interactive]`